sdk/storage/azure-storage-blob-changefeed/azure/storage/blobchangefeed/__init__.py

```diff
@@ -0,0 +1,14 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from azure.core.paging import ItemPaged
+from ._change_feed_client import ChangeFeedClient
+
+
+__all__ = [
+    'ChangeFeedClient',
+    'ItemPaged'
```

Comment:
We don't need to expose `ItemPaged` here - users are unlikely to need this type, and if they do they can import it from azure-core directly.